.travis.yml

```diff
@@ -58,8 +60,15 @@ before_install:
 
 install:
   - go get -u github.com/golang/dep/cmd/dep
+  # install reviewdog and golangci-lin
+  - curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s
+  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s latest
 
 before_script:
+  # static analyze
+  - bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number | bin/reviewdog -f=golangci-lint -level=error -reporter=github-pr-check
+  - bin/golangci-lint run -c=.golangci.yml --out-format=line-number | bin/reviewdog -f=golangci-lint -level=error -reporter=github-pr-review
+  - dep check
```

Comment:
@percona-csalguero is it ok that dep check is failing?

Reply:
we don't upload vendor dir in PT